bundles/org.openhab.ui.homebuilder/USAGE.md

```diff
@@ -1,18 +1,116 @@
-# Home Builder
+---
+layout: documentation
+title: Home Builder
+source: https://github.com/openhab/openhab-core/blob/master/bundles/org.openhab.ui.homebuilder/USAGE.md
+---
 
-> Boilerplate for the [Items](http://docs.openhab.org/configuration/items.html), [sitemap](http://docs.openhab.org/configuration/sitemaps.html) files and [HABPanel](http://docs.openhab.org/addons/uis/habpanel/readme.html) dashboard.
+{% include base.html %}
 
-## Items
+<!-- Attention authors: Do not edit directly. Please add your changes to the appropriate source repository -->
```

Comment:
What do you mean with this?

Reply:
How wil it sync?

Reply:
@martinvw I specifically asked for this header. It is needed for docs.openhab.org, I'll implement the syncing.

Eventually this header will not be needed because I can generate in docs. For now this is the easiest approach till ESH issues 4503 is settled.